Tools/baselineDef.h

```diff
@@ -40,6 +40,19 @@ class BaselineVessel
         std::string METPhiLabel = "metphi";
 
         std::string muonsFlagIDLabel = "muonsFlagMedium";
+        try
+        {
+          const double &temp  = tr.getVar<double>("genHT");
```

Comment:
In future, we need to group configs for different sample/version so that it won't be a mess. You can change this to ntpVersion. It should work too